lib/easybill-rest-client/api_client.rb

```diff
@@ -11,12 +11,14 @@ class ApiClient
     DEFAULT_RETRY_COOL_OFF_TIME = 15
     DEFAULT_TRIES = 5
     MAX_PAGE_SIZE = 1000
+    OPEN_TIMEOUT = 5
 
     def initialize(options = {})
       @api_key = options.fetch(:api_key)
       @retry_cool_off_time = options.fetch(:retry_cool_off_time, DEFAULT_RETRY_COOL_OFF_TIME)
       @tries = options.fetch(:tries, DEFAULT_TRIES)
       @logger = options.fetch(:logger, Logger.new($stdout))
+      @logger.formatter = log_formatter
```

Comment:
Are you sure this is a good idea? E.g. if I do `ApiClient.new(logger: Rails.logger)`, *all* my Rails logs will be formatted using the easybill prefix.